app/models/concerns/order_shipping_method.rb

```diff
@@ -18,4 +18,18 @@ def shipping_method
     return if shipments.empty?
     shipments.first.shipping_method
   end
+
+  # Finds the shipment shipping_rate for the given shipping_method_id and selects that shipping_rate
+  #
+  # @return [Shipment]
+  def select_shipping_method(shipping_method_id)
+    return if shipping_method_id.nil? || shipments.empty?
```

Comment:
[blank?](https://apidock.com/rails/v3.2.13/Object/blank%3F) might be what you need here.

Reply:
:+1: done